transformer/cloudfoundrytransformer.go

```diff
@@ -244,8 +250,38 @@ func (t *CloudFoundry) Transform(newArtifacts []transformertypes.Artifact, alrea
 	return nil, artifactsCreated, nil
 }
 
+// NormalizeForEnvironmentVariableName converts the string to be compatible for environment variable name
+func (t *CloudFoundry) normalizeForEnvironmentVariableName(envName string) string {
+	newName := common.DisallowedDNSCharactersRegex.ReplaceAllLiteralString(strings.ToLower(envName), "_")
```

Comment:
the env variable need for follow DNS regex. Lets create a new regex with things allowed in a env variable. Also use the regex in a way that we don't need to use tolower and immediately follow it by toupper. Also we can move this function to common pacakge

Reply:
Need some more clarification on this.